reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+
+See [Anbox Platform SDK](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#anbox-platform-sdk).
+
+<a name="anbox-streaming-sdk"></a>
+### Anbox Streaming SDK
+
+An SDK that allows the development of custom streaming clients, using JavaScript or C/C++.
+
+See [Anbox Streaming SDK](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#streaming-sdk).
+
+<a name="android-app"></a>
+### Android app
+
+An application for the Android mobile operating system, usually provided as APK. To distinguish Android apps from Anbox Cloud applications, this documentation refers to Android apps as "apps", not "applications".
+
+<a name="android-archive"></a>
+### Android Archive (AAR)
+
+A compiled version of an Android library that can be used as a dependency for an Android app module.
+
+See [Create an Android library](https://developer.android.com/studio/projects/android-library) in the Android developer documentation.
+
+<a name="adb"></a>
+### Android Debug Bridge (ADB)
+
+A command-line tool that is included in the Android SDK Platform-Tools package and that allows to connect to and communicate with an Android device from your computer.
+
+See [Android Debug Bridge (adb)](https://developer.android.com/studio/command-line/adb) in the Android developer documentation.
+
+<a name="apk"></a>
+### Android Package Kit (APK)
+
+The file format used to package apps for the Android operating system.
+
+<a name="appium"></a>
+### Appium
+
+An open-source test automation tool that can be used to test native, mobile and hybrid web applications on Android.
+
+See [the Appium website](http://appium.io/).
+
+<a name="application"></a>
+### Application
+
+One of the main objects of Anbox Cloud. An application encapsulates an Android app and manages it within the Anbox Cloud cluster.
+
+See https://discourse.ubuntu.com/t/about-applications/17760.
+
+<a name="application-container"></a>
+### Application container
+
+A container that is created when launching an application.
+
+See [Application containers vs. raw containers](https://discourse.ubuntu.com/t/about-containers/17763#application-vs-raw).
+
+<a name="application-manifest"></a>
+### Application manifest
+
+A file that defines the attributes of an Anbox Cloud application.
+
+See https://discourse.ubuntu.com/t/application-manifest/24197.
+
+<a name="base-container"></a>
+### Base container
+
+A temporary container that is used when bootstrapping an application. It is automatically deleted when the application bootstrap is completed.
+
+See https://discourse.ubuntu.com/t/about-containers/17763.
+
+<a name="boot-package"></a>
+### Boot package
+
+The package to launch in an application container once the system has booted.
+
+<a name="bootstrap-process"></a>
+### Bootstrap process
+
+The process that builds the application and synchronises it across all LXD nodes in the cluster when creating an application.
```

Comment:
```suggestion
The process that builds and optimises the application to run on Anbox Cloud
```
The bootstrap process doesn't technically sync an application on the whole cluster, LXD does that automatically when a new container is launched on a node (which is why the first container created for an application takes longer to boot)